backend/src/main/java/com/pokemon/pokedex/config/Config.java

```diff
@@ -0,0 +1,21 @@
+package com.pokemon.pokedex.config;
+
+import com.pokemon.pokedex.repositories.PokemonRepository;
+import com.pokemon.pokedex.services.PokemonsServices;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
```

Comment:
To be honest, never use `proxyBeanMethods` so I had to search. Pretty interesting, but I will enjoy explaining this in the interview for my knowledge.

Reply:
I just used it so i didn't have to configure db connections